board/safety/safety_toyota.h

```diff
@@ -189,7 +202,6 @@ const safety_hooks toyota_hooks = {
 
 static void toyota_nolimits_init(int16_t param) {
```

Comment:
Remove this function and nolimits_hooks?

Reply:
yep